Utils/test_upload_flow/create_test_branch.py

```diff
@@ -5,23 +5,25 @@
 import subprocess
 import time
 from pathlib import Path
-from typing import Union
 from git import GitCommandError, Head, Repo
 from zipfile import ZipFile
 from packaging.version import Version
 
 from Tests.scripts.utils import logging_wrapper as logging
 from Tests.scripts.utils.log_util import install_logging
+from Utils.github_workflow_scripts.utils import get_env_var
 
 versions_dict = {}
 pack_items_dict = {}
 changed_packs = set()
 
+GITLAB_SERVER_HOST = get_env_var('CI_SERVER_HOST', 'code.pan.run')  # disable-secrets-detection
+GITLAB_PROJECT_NAMESPACE = get_env_var('CI_PROJECT_NAMESPACE', 'xsoar')  # disable-secrets-detection
```

Comment:
When the migration is over we'd need to change this default value.
(Add this to the Phase_2 issue.)

In several places all over this PR.